content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Distributed Tracing With LightStep [𝑥]PM
+description: How to configure the proxies to send tracing requests to LightStep [𝑥]PM.
+weight: 11
+keywords: [telemetry,tracing,lightstep]
+---
+
+This task shows you how to configure Istio to collect trace spans and send them to LightStep [𝑥]PM.
+[𝑥]PM lets you analyze 100% of unsampled transaction data from large-scale production software to produce meaningful
+distributed traces and metrics that help explain performance behaviors and accelerate root cause analysis. For more
+information, visit [LightStep](https://lightstep.com).
```

Comment:
:). I would kill this ad bit.. May be state it as "Lightstep is a hosted version of "run-your-own-zipkin" with a several value added features. For more info..."